src/main/resources/static/css/academicCourseSectionForTeachersStyle.css

```diff
@@ -677,19 +549,31 @@ form {
     cursor: pointer;
     font-family: "Montserrat", sans-serif;
 }
+list {
+    width: 100%;
+    padding: 0px;
+    background-color: #555555;
+    height: 65vh;
+    margin-top: 10px;
+    overflow-y: scroll;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-direction: column;
+}
 
 .list {
     width: 100%;
     background-color: #555555;
     height: 65vh;
     overflow-y: scroll;
 }
-.selectTeacher {
+.seleurl-pathctTeacher {
     margin-top: 6px;
```

Comment:
Please, rename the div's class name seleurl-pathctTeacher to sth discriptive.I guess you meant selecturl-pathteacher.